development/workflow.rst

```diff
@@ -0,0 +1,668 @@
+You can do this from the command line:
+
+.. code-block:: bash
+
+   git config --global user.name "Your Name"
+   git config --global user.email "your_email@institution.edu"
+
+Likewise, in your `GitHub account email settings <https://github.com/settings/emails>`_, add your institution-hosted email.
+We recommend that you set this institutional email as your **Primary GitHub** email address.
+This step ensures that Git commits you make `directly on GitHub.com <https://help.github.com/articles/github-flow-in-the-browser/>`_ (such as quick documentation fixes) have proper authorship metadata.
+
+*See also*: :ref:`git-setup-best-practices`.
+
+.. _workflow-jira:
+
+Agile development with JIRA
+===========================
+
+We use `JIRA <https://jira.lsstcorp.org>`_ to plan, coordinate and report our work.
+Your Technical/Control Account Manager (T/CAM) is the best resource for JIRA usage within your local group.
+T/CAMs can consult the `Technical/Control Account Manager Guide <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=21397653>`_.
+This section provides a high-level orientation for everyday DM development work.
+
+.. _workflow-jira-concepts:
+
+Agile concepts
+--------------
+
+Issue
+   Issues are the fundamental units of work/planning information in JIRA.
+Story Points
+   Story points are how we estimate and account for time and effort.
+   One story point is an idealized half day of uninterrupted work by a competent developer.
+Velocity
+   No developer works a two story point day.
+   Communication overhead, review work, and other activities will invariably eat into your day.
+   *Velocity* is the fraction of a story point that you can reasonably achieve in a half day.
+   A common velocity in DM is 0.7, so that you nominally accomplish 1.4 story points in a day.
+   We do not track velocities for individual developers; each DM group shares a common velocity.
+   Ask your T/CAM.
+Epic
+   Epics are a special type of issue, created by T/CAMs, that guide your work over a six month **cycle** in pursuit of DM's development roadmap (`LDM-240 <http://ls.st/ldm-240>`_).
+   At the start of each cycle, your T/CAM will create an epic (or several) and allocate *story points* to that epic.
+   You don't work directly on an epic; rather you work on *stories* (below) that cumulatively accomplish the epic.
+
+.. _workflow-jira-issues:
+
+Tickets
+-------
+
+All development work is done on these three types of **JIRA issues** that are generically referred to as **tickets**:
+
+Story
+   Stories are issues associated with an epic.
+   That is, stories are for work that accomplish your main goals for a cycle.
+Bug
+   A bug is an emergent (not planned with an epic) ticket that fixes a fault in the software that exists on ``master``.
+   Bug tickets are not associated with an epic.
```

Comment:
This isn't completely true as they are typically assigned to an EPIC when SPs are claimed for resolving the bug. This may need more pedantic explanation.